packages/core/package.json

```diff
@@ -34,6 +34,7 @@
     "ajv": "^6.5.3",
     "body-parser": "^1.17.2",
     "cookie": "^0.4.0",
+    "keytar": "7.7.0",
```

Comment:
I confirmed that the `keytar` dependency is license compatible

Reply:
Do I need to create a CQ for this package?

Reply:
> Do I need to create a CQ for this package?

@vinokurig I don't think so, it falls under the following [use-case](https://github.com/eclipse-theia/theia/wiki/Registering-CQs#case-new-or-updated-npm-production-dependency). I confirmed that the dependency looks fine, you may double-check if you like following this process.